### PR TITLE
add 1Tb to couch2

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

**Review by snopoke — APPROVED** [review]

File: fab/inventory/icds-new [hunk 1]
```diff
@@ -304,8 +304,8 @@ hostname='couch1'
 10.247.164.14
 
 [couch2:vars]
-devices=["/dev/sdb"]
-partitions=["/dev/sdb1"]
+devices=["/dev/sdb","/dev/sdb"]
```
Comment (dannyroberts) [inline fab/inventory/icds-new]: Is this supposed to say the same thing twice? Or is this supposed to say `/dev/sdc` with a c?

Comment (dannyroberts) [reply fab/inventory/icds-new]: (pretty unfamiliar with our LVM setup conventions, so just wanted to double check)

**Review by snopoke — COMMENTED** [review]
Should definitely be sdc. Thanks @dannyroberts

**Comment by sravfeyn** [comment]
Thanks for the correction!

**Pull request merged**